### ISSUE TITLE
Adam optimiser

### ISSUE THREAD

**Issue by johngabra** [ciceklab/UTRGAN#3]
Hi all,
when i run the code why i get such an error?

> WARNING:absl:`lr` is deprecated, please use `learning_rate` instead, or use the legacy optimizer, e.g.,tf.keras.optimizers.legacy.Adam.

**Comment by sinabr** [comment]
This is not a problem. This is only a warning, and it is telling us to use a different "key" for the learning rate. You can keep using the same code. Please let me know if you have any other issues.
